fs/fs.go

```diff
@@ -0,0 +1,55 @@
+package fs
+
+import (
+	"os"
+
+	"github.com/spf13/afero"
+)
+
+// Fs interface contains all methods implemented in MemFs and OsFs
+type Fs interface {
+	afero.Fs
+	Chown(string, int, int) error
+	Link(string, string) error
+	Symlink(string, string) error
+}
+
+type MemFs struct {
+	afero.Fs
+}
+
+func NewMemFs() MemFs {
+	return MemFs{Fs: afero.NewMemMapFs()}
+}
+
+func (fs MemFs) Chown(string, int, int) error {
+	return nil
+}
+
+func (fs MemFs) Link(string, string) error {
+	return nil
+}
+
+func (fs MemFs) Symlink(string, string) error {
+	return nil
+}
+
+type OsFs struct {
+	afero.Fs
```

Comment:
I don't understand where the `afero.Fs` attribute is used. Why isn't it used in the various implemented methods?

Reply:
It's because afero.Fs is already implementing a  lot of methods, we're just adding a few here.

```
osfs := fs.NewOsFs()
osfs.Mkdir() <- using afero.Fs
```